packages/clients/test/sui/wallet.integration.test.ts

```diff
@@ -134,6 +134,8 @@ describe("ShinamiWallet", () => {
       beneficiary,
     );
     console.log("txDigest", txDigest);
+    sui.waitForTransaction({ digest: txDigest });
```

Comment:
This is related to [sc-3767](https://app.shortcut.com/shinami/story/3767/accommodate-deprecation-of-waitforlocalexecution-on-json-rpc) . We may want to revise this once we tackle that ticket.